crates/core_arch/src/aarch64/neon/generated.rs

```diff
@@ -24470,7 +24470,7 @@ pub fn vrsrad_n_u64<const N: i32>(a: u64, b: u64) -> u64 {
 #[doc = "[Arm's documentation](https://developer.arm.com/architectures/instruction-sets/intrinsics/vrsubhn_high_s16)"]
 #[inline]
 #[target_feature(enable = "neon")]
-#[cfg_attr(test, assert_instr(rsubhn2))]
+#[cfg_attr(test, assert_instr(rsubhn))]
```

Comment:
It would be nice to keep checking for rsubhn2 in the little-endian case since that would help catch regressions in LLVM codegen. Can this be made conditional on `target_endian`?